sklearn/utils/estimator_checks.py

```diff
@@ -528,10 +528,10 @@ def checks_generator():
     )
 
 
-def check_estimator(Estimator, generate_only=False):
+def check_estimator(estimator, generate_only=False):
```

Comment:
> The fix is to give a default value to estimator ? what should it be ?

Ah yes, we would also need a default value for `estimator`. I think it can be `None` for now as a sentinel. 

```python
def `check_estimator`(estimator=None, generate_only=False, Estimator="deprecated"):
    if Estimator != "deprecated":
        warnings.warn(...)
        estimator = Estimator
```